transforms/basic/info.js

```diff
@@ -11,8 +11,8 @@ const getInfo = (options = {}) => ({
     type: 'string',
     defaultValue: 'Add your description',
   }),
-  contact: options.contact ? getContact(options.contact) : {},
-  license: options.license ? license(options.license) : {},
+  contact: options.contact ? getContact(options.contact) : undefined,
+  license: options.license ? license(options.license) : undefined,
```

Comment:
Thanks for the feedback. We would like to merge this PR 👍. However, is it possible to change this to instead add undefined as default value just remove the key of the final object?

```js
...(options.contact ? { contact: getContact(options.contact) } : {}),
...(options.license ? { license: license(options.license) } : {}),
```